Modules/@babylonjs/react-native/package.json

```diff
@@ -28,15 +28,15 @@
     "semver": "^7.3.2"
   },
   "peerDependencies": {
-    "@babylonjs/core": "^6.14.0",
+    "@babylonjs/core": "^7.6.2",
```

Comment:
It doesn't seem like we want this change... this is saying BRN requires at least BJS 7.6.2, but we document the min as 5.53.1. Seems like this should actually be ^5.53.1, right?

Reply:
yes! I'll fix that!

Reply:
actually, because of non-float support in vertex buffer. bjs version will be >=7.7.1

Reply:
Really? So this is basically a breaking change, where if anyone wants to update BRN, they are going to have to update BJS all the way to 7.7.1 as well? @bghgary

Reply:
I've update BN to the latest and works fine with BJS 5.53.1